collectors/rest/salesforce/collector.json

```diff
@@ -32,13 +32,13 @@
         ],
         "enableHeader": true
       },
-      "loginUrl": "`<Your_login_URL>`",
+      "loginUrl": "`<Login URL| >`",
```

Comment:
the space between the `|` and the `>` isn't necessary here. Currently it'll wind up creating a description that is literally " ".
This'll be fixed shortly (see CRIBL-21854) so it ultimately doesn't matter, but could be slightly cleaner as `<Login URL|>`